tensorflow_addons/layers/__init__.py

```diff
@@ -21,3 +21,4 @@
 from tensorflow_addons.layers.python.maxout import Maxout
 from tensorflow_addons.layers.python.poincare import PoincareNormalize
 from tensorflow_addons.layers.python.wrappers import WeightNormalization
+from tensorflow_addons.layers.python.sparsemax import Sparsemax
```

Comment:
Could you please alphabetize these? We're still  working on our style guidelines